Make.inc

```diff
@@ -81,38 +81,23 @@ ifneq (,$(findstring MINGW,$(OS)))
 override OS=WINNT
 endif
 
-#keep these if statements separate
```

Comment:
I commented when I first committed this, but it disappeared after I squashed.
https://github.com/ginggs/openlibm/commit/49059f7ee6b9d72cdeb962bc34c14d56432ade07#commitcomment-14003490

There is a comment ```#keep these if statements separate```, but it would make sense to have a sane default, otherwise the clean target in Makefile:
```rm -fr $$dir/*.o $$dir/*.a $$dir/*.$(SHLIB_EXT)*; \```
evaluates to:
```rm -fr $$dir/*.o $$dir/*.a $$dir/*.*; \```
and cleans everything!

I noticed this was changed in openspecfun and it now has a default.
See: JuliaLang/openspecfun@e91925a